### PR TITLE
Fix Clickable List Cell

### LINKED ISSUE #163: [PE-D] Purpose of clicking and colour highlight is not documented
Clicking on rows in the various tables creates a highlight on the left. This highlight is not documented, unsure if it represents anything. Would be good if its clearly mentioned if its an aesthetic thing or has associated functionality

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: DrWala/ped#16

### PR DESCRIPTION
**Pull request by Assyarul**
The current ListView variables allows the user to select the
correspong list cells which we do not want.

Let's fix that.

Fixes #163 #162

### CODE REVIEW

File: src/main/java/seedu/address/ui/GroupListPanel.java [hunk 1]
```diff
@@ -25,6 +27,14 @@
      */
     public GroupListPanel(ObservableMap<Name, Group> groupMap) {
         super(FXML);
+        //Prevent mouse selection
```
Comment (ivantjh) [inline src/main/java/seedu/address/ui/GroupListPanel.java:31]: nitpick: can we standardise whether to leave a new line here and keep it the same throughout the UI components?

File: src/main/java/seedu/address/ui/PersonListPanel.java [hunk 2]
```diff
@@ -25,6 +27,13 @@
      */
     public PersonListPanel(ObservableList<Person> personList) {
         super(FXML);
+        //Prevent mouse selection
+        personListView.addEventFilter(MouseEvent.MOUSE_PRESSED, new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent event) {
+                event.consume();
+            }
+        });
```
Comment (ivantjh) [inline src/main/java/seedu/address/ui/PersonListPanel.java:36]: nitpick: lets keep it consistent with the other UI components and leave a new line here

**Review by ivantjh — APPROVED** [review]
lgtm! We probably need a new way to let the user know which group is selected (is possible)!

File: src/main/java/seedu/address/ui/PersonListPanel.java (hunk 2 above)
Comment (Assyarul) [reply src/main/java/seedu/address/ui/PersonListPanel.java:36]: >lgtm! We probably need a new way to let the user know which group is selected (is possible)!

We don't have to. The list cell can still be selected i.e the green vertical bar will still pop out when you type in the command like previously implemented.

**Review by chesterhow — CHANGES_REQUESTED** [review]
Can we also add a 10px padding to each GroupListViewCell?

Edit: its okay I'll handle it in my PR #221

File: src/main/java/seedu/address/ui/GroupListPanel.java (hunk 1 above)
Comment (chesterhow) [reply src/main/java/seedu/address/ui/GroupListPanel.java:31]: Agree with ivan that we should add a space after the `super(FXML)` line for consistency

File: src/main/java/seedu/address/ui/PersonListPanel.java (hunk 2 above)
Comment (ivantjh) [reply src/main/java/seedu/address/ui/PersonListPanel.java:36]: Ok, thats great!

**Review by chesterhow — APPROVED** [review]

**Pull request merged**